src/v/cluster/metrics_reporter.cc

```diff
@@ -430,6 +431,8 @@ void rjson_serialize(
         rjson_serialize(w, d);
     }
     w.EndArray();
+    w.Key("storage_space_alert");
```

Comment:
I think this isn't really useful as a part of metrics report. Maybe we can just drop this field

Reply:
Yep. I'd really prefer to avoid long lived feature branches and work iteratively versus putting everything in one PR. It adds extra hours of rebasing / rebuilding (my build script logs time spent) for zero benefit. Another reason is you can queue up 20 commits over two weeks and then someone might request late changes. Now you've wasted much more time. Smaller / faster PR cycles mitigates this.

Reply:
This is a good description http://www.extremeprogramming.org/rules/integrateoften.html

Reply:
i think that this commit can simply be dropped, no need to rebase

Reply:
As we discussed: metrics reporter is not really used for alerting or exceptional conditions (what I thought when I added the metric there). I'll remove that commit and force-push an update shortly.